test.js

```diff
@@ -6,13 +6,7 @@ import globby from '.';
 
 const cwd = process.cwd();
 const tmp = 'tmp';
-const fixture = [
-	'a.tmp',
-	'b.tmp',
-	'c.tmp',
-	'd.tmp',
-	'e.tmp'
-];
+const fixture = ['a.tmp', 'b.tmp', 'c.tmp', 'd.tmp', 'e.tmp'];
```

Comment:
Don't do unrelated changes. It's impossible to review this PR because of all the unrelated changes.